### PR TITLE
Feature: Added an option to copy the User ID from the about page

### PR DESCRIPTION
**Pull request by yaira2**
**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

**Copy User Id**
1. Opened about page
2. Confirmed Copy > User Id works as expected

**Report Bug**
- Reporting a bug from the about page automatically passes the user id to GitHub

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -48,11 +48,21 @@ body:
...
       placeholder: "e.g. 10.0.22621.1848"
     validations:
       required: true
 
+    # User ID
+  - type: input
+    id: user_id
+    attributes:
+      label: User Id
+      description: Your User Id is a guid that can be matched up with bug reports in Sentry. To copy your User Id, access it from Settings -> About -> Copy -> User Id.
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:61]:
```suggestion
      description: Your User Id is a GUID that can be matched up with bug reports in Sentry. To copy your User Id, access it from Settings -> About -> Copy -> User Id.
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -3944,4 +3944,7 @@
   <data name="Toolbars" xml:space="preserve">
     <value>Toolbars</value>
   </data>
+  <data name="UserId" xml:space="preserve">
+    <value>UserId</value>
```
Comment (0x5bfa) [inline src/Files.App/Strings/en-US/Resources.resw:3948]:
```suggestion
    <value>User ID</value>
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -48,11 +48,21 @@ body:
...
       placeholder: "e.g. 10.0.22621.1848"
     validations:
       required: true
 
+    # User ID
+  - type: input
+    id: user_id
+    attributes:
+      label: User Id
+      description: Your User Id is a GUID that can be matched up with bug reports in Sentry. To copy your User Id, access it from Settings -> About -> Copy -> User Id.
```
Comment (yaira2) [inline .github/ISSUE_TEMPLATE/bug_report.yml:61]:
```suggestion
      label: User ID
      description: Your User ID is a GUID that can be matched up with bug reports in Sentry. To copy your User ID, access it from Settings -> About -> Copy -> User ID.
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 4]
```diff
@@ -3944,4 +3944,7 @@
   <data name="Toolbars" xml:space="preserve">
     <value>Toolbars</value>
   </data>
+  <data name="UserId" xml:space="preserve">
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3947]:
```suggestion
  <data name="UserID" xml:space="preserve">
```

File: src/Files.App/Views/Settings/AboutPage.xaml [hunk 5]
```diff
@@ -42,6 +42,7 @@
 						<MenuFlyout>
 							<MenuFlyoutItem Command="{x:Bind ViewModel.CopyAppVersionCommand}" Text="{helpers:ResourceString Name=AppVersion}" />
 							<MenuFlyoutItem Command="{x:Bind ViewModel.CopyWindowsVersionCommand}" Text="{helpers:ResourceString Name=WindowsVersion}" />
+							<MenuFlyoutItem Command="{x:Bind ViewModel.CopyUserIdCommand}" Text="{helpers:ResourceString Name=UserId}" />
```
Comment (yaira2) [inline src/Files.App/Views/Settings/AboutPage.xaml:45]:
```suggestion
							<MenuFlyoutItem Command="{x:Bind ViewModel.CopyUserIDCommand}" Text="{helpers:ResourceString Name=UserID}" />
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 6]
```diff
@@ -176,12 +194,18 @@ public string GetWindowsVersion()
...
+		{
+			return GeneralSettingsService.UserId;
+		}
 
 		public string GetVersionsQueryString()
 		{
 			var query = System.Web.HttpUtility.ParseQueryString(string.Empty);
 			query["files_version"] = GetAppVersion();
 			query["windows_version"] = GetWindowsVersion();
+			query["user_id"] = GetUserId();
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:208]:
```suggestion
			query["user_id"] = GetUserID();
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 7]
```diff
@@ -176,12 +194,18 @@ public string GetWindowsVersion()
 		{
 			return SystemInformation.Instance.OperatingSystemVersion.ToString();
 		}
+		
+		public string GetUserId()
+		{
+			return GeneralSettingsService.UserId;
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:200]:
```suggestion
		public string GetUserID()
		{
			return GeneralSettingsService.UserID;
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 8]
```diff
@@ -161,6 +168,17 @@ public void CopyWindowsVersion()
 				Clipboard.SetContent(dataPackage);
 			});
 		}
+		
+		public void CopyUserId()
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:172]:
```suggestion
		public void CopyUserID()
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 9]
```diff
@@ -161,6 +168,17 @@ public void CopyWindowsVersion()
...
+		{
+			SafetyExtensions.IgnoreExceptions(() =>
+			{
+				DataPackage dataPackage = new DataPackage();
+				dataPackage.RequestedOperation = DataPackageOperation.Copy;
+				dataPackage.SetText(GetUserId());
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:178]:
```suggestion
				dataPackage.SetText(GetUserID());
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 10]
```diff
@@ -76,6 +82,7 @@ public AboutViewModel()
 
 			CopyAppVersionCommand = new RelayCommand(CopyAppVersion);
 			CopyWindowsVersionCommand = new RelayCommand(CopyWindowsVersion);
+			CopyUserIdCommand = new RelayCommand(CopyUserId);
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:85]:
```suggestion
			CopyUserIDCommand = new RelayCommand(CopyUserID);
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 11]
```diff
@@ -33,6 +38,7 @@ public PackageVersion AppVersion
 
 		public ICommand CopyAppVersionCommand { get; }
 		public ICommand CopyWindowsVersionCommand { get; }
+		public ICommand CopyUserIdCommand { get; }
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:41]:
```suggestion
		public ICommand CopyUserIDCommand { get; }
```

File: src/Files.App/ViewModels/Settings/AboutViewModel.cs [hunk 12]
```diff
@@ -176,12 +194,18 @@ public string GetWindowsVersion()
 		{
 			return SystemInformation.Instance.OperatingSystemVersion.ToString();
 		}
+		
+		public string GetUserID()
+		{
+			return GeneralSettingsService.UserID;
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/AboutViewModel.cs:200]:
```suggestion
			return GeneralSettingsService.UserId;
```

**Review by 0x5bfa — APPROVED** [review]
Code wise, LGMT.

**Pull request merged**